data-processing/append_mortality_data.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+
+
+class AppendMortalityData:
+
+    def __init__(self):
+        from pyspark.sql import SparkSession
+        import os
+
+        self.psql_user = os.environ['POSTGRESQL_USER']
+        self.psql_pw = os.environ['POSTGRESQL_PASSWORD']
+
+        self.spark = SparkSession \
+            .builder \
+            .appName("Write mortality data to DB") \
+            .getOrCreate()
+
```

Comment:
Likewise this should be a separate class/method. 

There's a lot of different opinion on how to best work with the `spark` variable. Two popular approaches I can think of: 
1) If classes/methods/functions need to use the `spark` variable directly (e.g. `spark.read.parquet()` you can just make it an actual variable like `def my_function(inputs, _spark)` and then plug in an actual `spark` variable when you execute the function in `main()`. 

2) at the top of a file which needs to use `spark` you could do something like `_spark = SparkSession.active` or even just `_spark = Sparksession.builder.getOrCreate()` and have it only use the "get" part. So basically then you'll have some main `spark` variable, and when the functions/methods start getting called the variable `_spark` they use will just get the active session (which is `spark`). This is more or less what you're doing here, but the variable is set at the to of the file instead of as a separate method.